source/fundamentals/auth.txt

```diff
@@ -0,0 +1,384 @@
+your connection string and pass it to the ``MongoClients.create()`` method
+to instantiate your ``MongoClient``. For descriptions of how to use a
+connection string to specify your authentication method, navigate to the
+relevant mechanism section and select the :guilabel:`Connection String` tab.
+
+The ``MongoCredential`` class contains static factory methods that allow you
+to specify your authentication mechanism and credentials. You can use this
+helper class to keep your code organized. If you choose to use the
+``MongoCredential`` helper class, you need to use the
+``MongoClientSettings.Builder`` class to configure your connection settings
+when constructing your ``MongoClient``. For descriptions of how to use the
+``MongoCredential`` class to specify your authentication method, navigate
+to the relevant mechanism section and select the :guilabel:`MongoCredential`
+tab.
+
+For more information on the aforementioned classes and methods, refer to the
+following API documentation:
+
+- :java-sync-api:`MongoClients.create() </com/mongodb/client/MongoClients.html#create(com.mongodb.ConnectionString)>`
+- :java-sync-api:`MongoClient </com/mongodb/client/MongoClient.html>`
+- :java-sync-api:`MongoClientSettings.Builder </com/mongodb/MongoClientSettings.Builder.html>`
+- :java-sync-api:`MongoCredential </com/mongodb/MongoCredential.html>`
+
+Mechanisms
+----------
+
+.. _default-auth-mechanism:
+
+Default
+~~~~~~~
+
+The default authentication mechanism is a fallback setting that instructs
+the driver to negotiate the first authentication mechanism supported by the
+server in the following order of preference:
+
+#. ``SCRAM-SHA-256``
+#. ``SCRAM-SHA-1``
+#. ``MONGODB-CR``
+
+When you use the default authentication mechanism, the driver first attempts
+to authenticate using ``SCRAM-SHA-256``. If the version of the MongoDB
+instance does not support that mechanism, the driver attempts to authenticate
+using ``SCRAM-SHA-1``. If the instance does not support that mechanism either,
+the driver attempts to authenticate using ``MONGODB-CR``.
+
+The following code snippets show how to specify the connection mechanism,
+using the following placeholders:
+
+* ``username`` - your MongoDB username
+* ``password`` - your MongoDB user's password
+* ``hostname`` - network address of your MongoDB server, accessible by your client
+* ``port`` - port number of your MongoDB server
+* ``authenticationDb`` - MongoDB database that contains your user's
+  authentication data. If you omit this parameter, the driver uses the
+  default value ``admin``.
+
+Select the :guilabel:`Connection String` or the :guilabel:`MongoCredential`
+tab below for instructions and sample code for specifying this authentication
+mechanism:
+
+.. tabs::
+
+   .. tab::
+      :tabid: Connection String
+
+      To specify the default authentication mechanism using a connection
+      string, explicitly omit the mechanism. Your code to instantiate
```

Comment:
It's weird to say to explicitly omit something.  Seems like an oxymoron!.  Can we just remove the word `explicitly`?

Reply:
Sure, makes sense!